pkg/webui/components/form/field/field.styl

```diff
@@ -33,6 +33,9 @@
       width: 14rem
       font-weight: $fw.normal
 
+  .glossary-link
```

Comment:
Is this necessary?

Reply:
Looking at #3162 this class was added based on this particular [comment](https://github.com/TheThingsNetwork/lorawan-stack/pull/3162#discussion_r493672041), I only added `line-height` property to it.